pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -83,6 +83,7 @@ func NewMustGatherCommand(f kcmdutil.Factory, streams genericclioptions.IOStream
 	cmd.Flags().StringSliceVar(&o.ImageStreams, "image-stream", o.ImageStreams, "Specify an image stream (namespace/name:tag) containing a must-gather plugin image to run.")
 	cmd.Flags().StringVar(&o.DestDir, "dest-dir", o.DestDir, "Set a specific directory on the local machine to write gathered data to.")
 	cmd.Flags().StringVar(&o.SourceDir, "source-dir", o.SourceDir, "Set the specific directory on the pod copy the gathered data from.")
+	cmd.Flags().Int64Var(&o.Timeout, "Timeout", 10, "Set a specific timeout to use - by default a 10 minutes will be used")
```

Comment:
Flags are always lower case. 
Description: `The length of time to gather data, in seconds. Defaults to 10 minutes."
Always use seconds, to be consistent with everything else.